.tekton/collector-pull-request.yaml

```diff
@@ -0,0 +1,391 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/stackrox/collector?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "500"
+    # TODO(ROX-21073): re-enable for all PR branches
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && (source_branch.contains("rhtap") || source_branch.contains("konflux"))
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: acs
+    appstudio.openshift.io/component: collector
+    pipelines.appstudio.openshift.io/type: build
+  name: collector-on-pull-request
+  namespace: rh-acs-tenant
+
+spec:
+
+  params:
+  - name: dockerfile
+    value: collector/container/konflux.Dockerfile
+  - name: git-url
+    value: '{{repo_url}}'
+  - name: image-expires-after
+    value: '13w'
+  - name: output-image
+    value: quay.io/redhat-user-workloads/rh-acs-tenant/acs/collector:on-pr-{{revision}}
+  - name: path-context
+    value: .
+  - name: revision
+    value: '{{revision}}'
+  - name: rebuild
+    value: 'true'
+  # TODO(ROX-20234): Enable hermetic builds
+  # - name: hermetic
+  #   value: "true"
+  # No language dependencies are required for central-db image.
+  - name: prefetch-input
+    value: ''
+
+  workspaces:
+  - name: workspace
+    volumeClaimTemplate:
+      metadata:
+        creationTimestamp: null
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 5Gi
+      status: { }
+  - name: git-auth
+    secret:
+      secretName: '{{ git_auth_secret }}'
+
+  timeouts:
+    pipeline: 1h30m0s
```

Comment:
1. This setting is absent from `collector-push.yaml`. Please add it there if it's meaningful.
2. If the timeout is deliberate, please add a short comment explaining it (e.g. we increase it because the default one is not enough to account for downloading the support package). Otherwise, if you introduced it as part of testing and the default one is also ok, I suggest removing it.

Reply:
Removed. There was a time where the pipeline regularly took more than 1 hour, but that has not happened in the last 7 days.